VTEX - Pick and Pack API.json

```diff
@@ -0,0 +1,4196 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+        "title": "Pick and Pack API",
+        "description": "The Pick and Pack API provides comprehensive functionality for order management, picking operations, shipment tracking, and returns processing. The main capabilities include:\n\n- Retrieving detailed order information including items, packages, and status\n\n- Accessing worksheet data for picking operations\n\n- Querying shipment tracking and delivery details\n\n- Getting facility information",
+        "contact": {},
+        "version": "1.0"
+    },
+    "servers": [
+        {
+            "url": "https://auth.pickingnpacking.com/prod/{environment}",
+            "variables": {
+                "environment": {
+                    "default": "DefaultParameterValue",
+                    "description": "Server to access the authentication necessary to make a request in the Pick and Pack Provider API."
+                }
+            }
+        },
+        {
+            "url": "https://api.pick-and-pack.com/prod/v1/{environment}",
+            "variables": {
+                "environment": {
+                    "default": "DefaultParameterValue",
+                    "description": "Server to access the authentication necessary to make a request in the Pick and Pack Provider API."
+                }
+            }
+        }
+    ],
+    "paths": {
+        "/token": {
+            "post": {
+                "tags": [
+                    "Authentication"
+                ],
+                "summary": "Create JWT token",
+                "description": "Creates a `JWT` (JSON Web Token) necessary to access the Pick and Pack Last Mile Protocol API information.\r\n\r\n>ℹ️ To generate the `JWT` it is necessary to have the `API KEY` previously generated by the Pick and Pack team.\r\n\r\n## Permissions\r\n\r\nThis endpoint does not require [permissions](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3).",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    }
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/CreatetokenRequest"

```

Comment:
Not blocking the PR over this, but I don't think it is a good practice to use ref for a component that is used only once in the file. It usually makes maintenance more difficult.